### DISCUSSION TITLE
Lesson 6: (5:45:55) AttributeError: 'function' object has no attribute 'transfer

### DISCUSSION THREAD

**Discussion by santoroeth** [smartcontractkit/full-blockchain-solidity-course-py#1237] [Q&A] [answered]
Hello, I have this problem when trying to run the "fund_and_withdraw.py" script, after having defined the "fund" function,any help would be great!

![image](https://user-images.githubusercontent.com/97750911/157454989-bed4d916-9fea-46e8-ae81-3cc3133611fa.png)

reviewing the "FundMe.sol" contract, from the beginning I have had this warning, but I had not had any problems with it

![image](https://user-images.githubusercontent.com/97750911/157455239-233ef037-36df-491e-a927-05a192f69131.png)

![image](https://user-images.githubusercontent.com/97750911/157455292-79750bed-1e01-4380-938c-124fb1e2535a.png)

this is my "fund_and:withdraw"

![image](https://user-images.githubusercontent.com/97750911/157455453-ac4a3575-131b-44a8-83d4-0f0b4d823990.png)

this is my "helpful_scripts"

![image](https://user-images.githubusercontent.com/97750911/157455568-e8411311-afc6-4dd1-95b5-ea542c7e9601.png)

this is my "brownie-config"

![image](https://user-images.githubusercontent.com/97750911/157455664-7eefd449-6bfd-48ec-881f-ad0320b86f13.png)

and my "FundMe.sol"

![image](https://user-images.githubusercontent.com/97750911/157455821-88bb9fdd-fddf-4c09-ac78-2267a69ea510.png)
![image](https://user-images.githubusercontent.com/97750911/157455905-f93b6327-152e-40b7-93ec-82867e0c2675.png)
![image](https://user-images.githubusercontent.com/97750911/157456022-73c3c709-5379-43fa-8a06-ea2029bafe8c.png)
![image](https://user-images.githubusercontent.com/97750911/157456092-ac8daed6-202a-4528-9d50-10a950e8ffc4.png)

sorry for all the notes on the code, i think it looks a bit disorganized, but i'm trying to understand as much as i learn

Any help would be great, thanks!

**Comment by santoroeth** [comment, marked as answer]
I'm sorry I already solved it,

I had a syntax problem on this line --> account = get_accounts()

**Reply by cromewar** [reply]
Hello @santoroeth, sorry for the delay.
Nice catch!!
